examples/framework/preferences/localizationPreference/urlPath/fr/index.html

```diff
@@ -0,0 +1,105 @@
+        <script type="text/javascript" src="../../../../../../src/framework/enhancement/js/ProgressiveEnhancement.js"></script>
+
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/URLUtilities.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/Store.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/UIEnhancer.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/ArrowScrolling.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/PrefsEditor.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/Panels.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/LocalizationPanel.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/Enactors.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/LocalizationEnactor.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/SeparatedPanelPrefsEditor.js"></script>
+        <script type="text/javascript" src="../../../../../../src/components/slidingPanel/js/SlidingPanel.js"></script>
+        <script type="text/javascript" src="../../../../../../src/components/switch/js/Switch.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/PrimaryBuilder.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/Builder.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/AuxBuilder.js"></script>
+        <script type="text/javascript" src="../../../../../../src/framework/preferences/js/LocalizationSchemas.js"></script>
+
+        <script type="text/javascript" src="../js/localization.js"></script>
+    </head>
+
+    <body class="fl-focus">
+
+        <!-- BEGIN markup for Preferences Editor -->
+        <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the mobile view.
+                A separate panel bar for mobile displays is needed to preserver the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-smallScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
```

Comment:
"Reset" should be "Réinitialiser" or "Remettre"

Reply:
@BlueSlug what do you think would be better to use in this case?